apps/web/app/courses/[courseCode]/page.tsx

```diff
@@ -37,6 +38,14 @@ export default function CourseReview({ params }: { params: { courseCode: string
 			return;
 		}
 
+		if (result.title === "Success!") {
```

Comment:
Now that I check this, the title doesn't really return as "Success!".

- For `submitNewReview`, the title will be `Submit review successfully` 
Ref: https://github.com/stamford-syntax-club/course-compose/blob/main/apps/web/lib/api/api.ts#L105

- For `submitEditedReview`, the title will be `Submitted your edited review successfully`
Ref: https://github.com/stamford-syntax-club/course-compose/blob/main/apps/web/lib/api/api.ts#L167

Consider checking with other attributes that represent the success status.